### DISCUSSION TITLE
Is there a way to import pkgs as part of `cue eval -e`

### DISCUSSION THREAD

**Discussion by cdhunt** [cue-lang/cue#1622] [Q&A] [answered]
I'm trying to run `cue eval myfile.json -e` and the expression includes `regexp.FindSubmatch`. This works fine if I do the add the expression to a cue file with the proper imports at the top, but I'm testing to see if I can bypass generating a cue file and basically use cue in place of jq.

Is there a way with the cue cli to inject the import statements as flags or in the expression string?

[SO Question](https://stackoverflow.com/questions/71788460/is-there-a-way-to-import-pkgs-as-part-of-cue-eval-e)

**Comment by mpvl** [comment, marked as answer]
Which version are you using? The command line flags, as a convenience and to work around this issue, infer unambiguous imports automatically.  It won't work, though, if the package name occurs as a field at the top level in the configuration.

**Reply by cdhunt** [reply]
cue version v0.4.3 windows/amd64

I think it's a parsing issue with PowerShell. When I triple quote the regex pattern it does work.
